docs/emulation.md

```diff
@@ -0,0 +1,32 @@
+
+# Emulation in Lighthouse
+
+In Lighthouse, "Emulation" refers to the screen/viewport emulation and UserAgent string spoofing.
+["Throttling"](./throttling.md) covers the similar topics around network connection throttling/simulation and CPU throttling.
+
+In Lighthouse v7, most of the configuration regarding emulation changed to be more intuitive and clear. The [tracking issue](https://github.com/GoogleChrome/lighthouse/issues/10910
```

Comment:
there's something about this sentence being *in the PR that brings those updates* being really funny to me 😆 

I love it. Feels odd as the 2nd sentence for people just stumbling onto the generic emulation docs though, move down to "Changes in v7"?